controllers/utils/validate.ts

```diff
@@ -0,0 +1,37 @@
+import emailValidator from 'email-validator';
+
+export const email = (email: any) => {
+    return emailValidator.validate(email);
+}
+
+export const noneIsBlank = (Obj: any, neededBodyKeys: any) => {
+
+    for(let key of neededBodyKeys) {
+        if (String(Obj[key]).length == 0) {
+            return false;
+        }
+    }
+    return true;
+}
+
+export const countOfKeysMatch = (Obj: any, neededKeys: any) => {
+
+    const countOfKeys = Object.keys(Obj).length;
+    return neededKeys.length <= countOfKeys; 
+

```

Comment:
can we make sure that we don't use the type: `any` ?
Also, can we add a prettier/linter so that we don't have unnecessary whitespace and linebreaks?